samples/subsys/nvs/boards/yd_esp32.conf

```diff
@@ -0,0 +1 @@
+CONFIG_HEAP_MEM_POOL_SIZE=256
```

Comment:
I think this .conf is not needed anymore (any esp32 board).

Reply:
Done. Sample output after removal:

![yd_esp32_nvs_sample](https://github.com/zephyrproject-rtos/zephyr/assets/103332557/f3fbb652-88ab-4a13-a8db-40a5b6a9b4b2)